src/components/molecules/CameraMicrophoneControls/CameraMicrophoneControls.tsx

```diff
@@ -11,6 +11,22 @@ import { VideoOverlayButton } from "components/atoms/VideoOverlayButton";
 
 import "components/molecules/CameraMicrophoneControls/CameraMicrophoneControls.scss";
 
+const setTrackStateTo = (enable: boolean) => ({
+  track,
+}: AudioTrackPublication | VideoTrackPublication) => {
+  if (!track) {
+    return;
+  }
+  if (enable && "enable" in track) {
+    track.enable();
+    return;
+  }
+  if ("disable" in track) {
+    track.disable();
+    return;
+  }
+};
+
```

Comment:
Yep, TS had issues with `?.` and `?.()` so good enough on the `in` operator.

Still generating a function, but now with no nesting of `if`s, concise and linear code for human readability.